src/Settings.ts

```diff
@@ -20,16 +19,16 @@ type Glob = string;
 // in favor of `@jest/types`, which exports the full config interface.
 
 type ProjectConfiguration = {
```

Comment:
In principle we could also use interfaces, but since there's hardly any difference, it's probably better to stick to the already existing types.